packages/steps/src/LionSteps.js

```diff
@@ -56,8 +56,13 @@ export class LionSteps extends ObserverMixin(LionLitElement) {
   }
 
   get steps() {
-    let steps = this.children;
-    steps = Array.from(steps).filter(step => step.nodeName === 'LION-STEP');
+    const slots = this.shadowRoot.querySelectorAll('slot');
+    const slot = Array.from(slots).filter(element => element.name === '')[0];
+    if (!slot) {
+      return [];
+    }
```

Comment:
there is no situation where you don't have a default slot, this is an internal requirement of your own component which is guaranteed, no need to do defensive programming here

Reply:
changed the whole method implementation to a simplistic one, so it is no longer applicable